index.html

```diff
@@ -103,11 +98,8 @@ <h3>Documentation</h3>
     <div class="wrapper">
       <h2>Support</h2>
       <p>
-        Open ONI is currently not ready for general production use outside the
-        institutions who have been able to provide time for developer
-        participation and have deployed it individually. Documentation and
-        changes to facilitate more standardized deployment are under active
-        development, but maintainers can only provide a minimal amount of
+        Open ONI is excited to announce the production-ready launch of release
+        1.0. However, maintainers can only provide a minimal amount of
```

Comment:
This doesn't seem like the right place for this text. I think we can just drop it to start with `Maintainers can only provide …`

Reply:
I think there should be somewhere that we list the current stable release, so perhaps we can add that back in somewhere when 1.0 launches

Reply:
That's reasonable. I think we'd probably want to list it in the Install area. I did add a note to my release checklist to update these pages, so hopefully it will get updated quickly after each new release.

Reply:
I think you could add 0.13 there for now so we have it in place :)